Makefile

```diff
@@ -85,7 +85,13 @@ upgrade-db: upgrade-pip install-backend
 	export PYTHONPATH=./backend && \
 	alembic -c backend/alembic.ini upgrade head
 
-clean:
```

Comment:
Is this intended? Even if `clean` is not used in the pipelines it is a useful command for local development

Reply:
Oh, I think I missed it during merge. There were some 'removing edits' from main.
Just a sec, I get it back